src/post.js

```diff
@@ -10,12 +10,42 @@ module.exports = async config => {
   const name = `v${pkg.version}`;
   const tag = {owner, repo, ref: `refs/tags/${name}`, sha: await promisify(gitHead)()};
   const body = await promisify(plugins.generateNotes)(config);
-  const release = {owner, repo, tag_name: name, name, target_commitish: branch, draft: !!debug, body};
+  let githubPreRelease = false;
+  let release = {
+    owner,
+    repo,
+    tag_name: name,
+    name,
+    target_commitish: branch,
+    draft: !!debug,
+    body,
+    prerelease: githubPreRelease,
+  };
 
   if (debug && !githubToken) {
```

Comment:
maybe this should come after deciding whether it's a pre-release or not

Reply:
since we are still deciding the value of a release property

Reply:
Yes :) I think made a review comment in that direction.